config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-periodics-main.yaml

```diff
@@ -14,12 +15,20 @@ periodics:
     - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230513-7e1db2f1bb-1.25
       command:
       - "./scripts/ci-test.sh"
+      resources:
+        requests:
+          cpu: "1"
+          memory: "2Gi"
+        limits:
+          cpu: "1"
+          memory: "2Gi"
```

Comment:
@rjsadow @dims are these resource quotas standard and numbers fine? Do we need to increase anything, perhaps?

Reply:
It's hard to say as each job is different. It's going to be best to work with the CAPI maintainers to find an initial set of values.

It'll be important that someone is watching the jobs for a while after this merges to see if jobs start failing from respurce backoff. I imagine most changes we have for this larger effort will go through some itterations.

Reply:
/hold 
until we have a generic resource quota numbers for the jobs that did not have them from the beginning